File: Maths/moyenne.js
/**
 * Fonction qui calcule la moyenne d'un tableau de nombres
 * @param {Array} arr - Tableau de nombres
 * @return {Number} Moyenne
 */
const moyenne = (arr) => arr.reduce((acc, cur) => acc + cur, 0) / arr.length;

// Test
console.log(moyenne([1, 3, 3, 7])); // 3.5

File: Python/Maths/moyenne.py
def moyenne(arr: list) -> float:
    """
    Description:
        Fonction qui calcule la moyenne d'un tableau.

    Paramètres:
        arr: {list} -- Tableau à traiter

    Retourne:
        {float} -- Moyenne du tableau

    Exemple:
        >>> moyenne([1, 3, 3, 7])
        3.5
    """
    return sum(arr) / len(arr)
